source/mongodb-handshake/handshake.rst

```diff
@@ -52,22 +53,43 @@ Specification
 Connection handshake
 --------------------
 
-The ``isMaster`` handshake MUST be performed on every socket to any and all servers
+MongoDB uses the ``hello`` or ``isMaster`` commands for handshakes and topology
+monitoring. ``hello`` is the modern and preferred command. ``isMaster`` is referred
+to as "legacy hello" and is maintained for backwards compatibility with servers
+that do not support the ``hello`` command.
+
+NOTE: The legacy hello command name is ``isMaster``, but servers will accept the
+alternate casing ``ismaster``. Other case variations result in ``CommandNotFound``.
+Drivers MUST take this case variation into account when determining which commands
+to encrypt, redact, or otherwise treat specially.
+
+If Versioned API is requested, drivers MUST use the ``hello`` command for the
```

Comment:
I suggest linking to the versioned API spec. Since drivers specify a `serverApi` class, I think the preferred term would be "server API version".

```suggestion
If a `server API version <../versioned-api/versioned-api.rst>`__ is requested, drivers MUST use the ``hello`` command for the
```